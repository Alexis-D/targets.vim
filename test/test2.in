function f() {
    command
}

int a = 5;
int b = a > 0
        ? 1
        : 2;

string x = `line 1
line 2
line 3`;

