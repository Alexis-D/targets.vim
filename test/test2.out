function f() {
foo
}

int a = 5;
foo;

string x = `foo`;

